check_missing_installations.sh

```diff
@@ -40,6 +47,30 @@ exit_code=${PIPESTATUS[0]}
 
 ok_msg="Command 'eb --missing ...' succeeded, analysing output..."
 fail_msg="Command 'eb --missing ...' failed, check log '${eb_missing_out}'"
+if [ "$exit_code" -eq 1 ] && [ ! -z $pr_exceptions ]; then
```

Comment:
Should this be `-ne 0` (EasyBuild but `exit 1` now, but it may be different in the future, see https://github.com/easybuilders/easybuild-framework/issues/4426)

```suggestion
if [ "$exit_code" -ne 0 ] && [ ! -z $pr_exceptions ]; then